TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+##Treaty of Vienna## 
+
+country_event = {
+	id = 2000000000
+	title = "EVTNAME2000000000"
+	desc = "EVTDESC2000000000"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+		name = "EVTOPTA2000000000"
+
+                set_global_flag = treaty_of_vienna_hungary
+
+		633 = { add_core = GER }
+		624 = { add_core = GER }
+		3286 = { add_core = GER }
+		623 = { add_core = GER }
+                3282 = { add_core = GER }

```

Comment:
Maybe group these like:
633 = { add_core = GER secede_province = GER }
for clarity and readability.
Also your indentation is not consistent. Please use tabs instead of spaces.